sdk/template/Azure.Template/perf/MiniSecretClientTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Identity;
+using Azure.Test.Perf;
+using CommandLine;
+
+namespace Azure.Template.Perf
+{
+    public class MiniSecretClientTest : PerfTest<MiniSecretClientTest.MiniSecretClientOptions>
+    {
+        private readonly MiniSecretClient _miniSecretClient;
+
+        public MiniSecretClientTest(MiniSecretClientOptions options) : base(options)
+        {
+            var keyVaultUri = GetEnvironmentVariable("KEYVAULT_URL");
```

Comment:
As a follow up we need to reuse the TestEnvironment infrastructure we have that allows reading encrypted files produced by New-TestResources. https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

Reply:
I don't understand this.  Is it to integrate with the automatic resource deployment (which we are planning to use for the perf/stress automation)?

Reply:
Yes, we have test resource infrastructure (scripts/utility classes) for live tests it would be nice to reuse  patterns that developers are already aware of for stress/perf as well.

Reply:
I will add this in a later PR.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/16708